scipy/io/_fast_matrix_market/src/_fmm_core.cpp

```diff
@@ -2,7 +2,8 @@
 // Use of this source code is governed by the BSD 2-clause license found in the LICENSE.txt file.
 // SPDX-License-Identifier: BSD-2-Clause
 
-#include <Python.h>
```

Comment:
This doesn't look correct, since there is direct usage of Python C API in this file. The linter also seems happy with the original version of the file. Unless I'm missing something here, I think you can revert this?

Reply:
`_fmm_core.hpp` is included next, and includes `Python.h` first, which the checker script knows.  Either way would work.

Reply:
I think the rationale for removing this where I kept the others was that I was the one to add this include, and recently.

Reply:
Ah okay, fair enough. I think it's technically more correct to keep it, but both work - whatever you prefer.